test/fixtures/core.scale/label-offset-vertical-axes.json

```diff
@@ -0,0 +1,40 @@
+{
+    "tolerance": "0.008",
```

Comment:
@simonbrunel I tried using long labels but then the test was failing even at high tolerance due to different font rendering on browsers/platform. So I tried replacing text with unicode shapes instead and it's working fine for me locally on firefox and chrome (macOS 10.2.3) but still failing on travis. Moreover, I can't see the build log for the latest commit. :/